src/js/aframe/aframeHelper.js

```diff
@@ -0,0 +1,42 @@
+const aframeHelper = {
+  async init() {
+    await this.getData();
+  },
+
+  async getData() {
+    const data = await storeChild.getAll();
+    const { nodes, links } = this.transformData(data);
+
+    return {
+      nodes,
+      links
+    };
+  },
+
+  transformData(data) {
+    const nodes = [];
+    let links = [];
+    for (const website in data) {
+      const site = data[website];
+      data[website].id = site.hostname;
+      data[website].name = site.hostname;
+      if (site.thirdParties) {
+        const thirdPartyLinks = site.thirdParties.map((thirdParty) => {
+          return {
+            source: website,
```

Comment:
is it possible that there will be duplicate link items? do we want to make the list contain only unique values - or is this fine?